src/components/encounter/ohri-encounter-form.tsx

```diff
@@ -35,6 +35,7 @@ interface OHRIEncounterFormProps {
   encounterDate: Date;
   provider: string;
   location: { uuid: string; name: string };
+  visitUUID: string;
```

Comment:
Do you want to make this an optional prop?

Reply:
This is supposed to be an optional param. On a side note, since we may need to pick a few props from the visit, why don't we just pass the current visit object? We are most likely to have a visit object within the workspace context, that way the engine won't have to refetch it. Thought?? @donaldkibet, @denniskigen

Reply:
i can pass the whole visit